pandas/core/groupby.py

```diff
@@ -1382,7 +1382,7 @@ def ngroups(self):
     def recons_labels(self):
         comp_ids, obs_ids, _ = self.group_info
         labels = (ping.labels for ping in self.groupings)
-        return decons_obs_group_ids(comp_ids, obs_ids, self.shape, labels)
+        return decons_group_ids(comp_ids, obs_ids, self.shape, labels, True)
```

Comment:
it's a little easier to read if you include argument names when supplying boolean, e.g., use `xnull=True`